src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1099,6 +1099,7 @@ class DocBaseViewer extends BaseViewer {
 
         if (this.areNewAnnotationsEnabled() && this.hasAnnotationCreatePermission()) {
             this.annotationControls.init({
+                fileId: this.options.file.id,
```

Comment:
Is it possible for this value to change after `init` has been called?

Reply:
If file id changes, the preview will be destroyed and re-init everything, right?

Reply:
Is that right?